app/views/spaces/index.html.erb

```diff
@@ -2,17 +2,32 @@
 <html lang="en" dir="ltr">
   <head>
     <meta charset="utf-8">
-    <title></title>
+    <title>Spaces</title>
+    <link rel="stylesheet" type="text/css" href="showlisting.css">
+    <link rel="stylesheet" href="../../assets/stylesheets/spaces.css">
   </head>
   <body>
-    <h1>Look at all the places!</h1>
 
-    <% @spaces.each do |space| %>
-    <%= "#{space.name}" %><br />
-    <%= "#{space.description}" %><br />
-    <%= "£#{space.price} per night" %><br />
-    <br />
-    <br />
-    <% end %>
+    <div class="container">
+      <div class="top1"><a class="navback" href="LASTPAGE"><img src="arr_backward.png"></a></div>
+      <div class="top2"><h1 class="heading">MakersBnB</h1> </div>
+      <div class="top3"><a class="navnext" href="NEXTPAGE"><img src="arr_forward.png"></a></div>
+
+
+      <% @spaces.each do |space| %>
+      <div id="card1" class="cardbg">
```

Comment:
Should we not increment the card number against the array location so each card has a unique card1, card2, card3 etc.? Not sure how this affects the css but might give more granularity over grid sections later.